exportReport/Utility/SpringUtilities.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 1995, 2008, Oracle and/or its affiliates. All rights reserved.
```

Comment:
What about wrong with it? I am just using the class that they wrote on their tutorial website, i didn't write it myself and there isn't an import for it. They had their own licensing already there, i didn't add anything to it. Did you want me to remove it?